.github/workflows/main.yml

```diff
@@ -44,8 +44,8 @@ jobs:
       - uses: actions/checkout@v2
       - uses: microsoft/playwright-github-action@v1
       - run: npm install
-      - run: npx aegir test -t browser --bail --cov
-      - run: npx aegir test -t webworker --bail
+      - run: npx aegir test -t browser --bail --cov --files test/**/*.browser.spec.{js,cjs,mjs}
```

Comment:
All changes in this file needs to happen because of: https://github.com/ipfs/aegir/issues/1097 Also, the test files had to be renamed because of this.